src/Generator/GoogleShopping.php

```diff
@@ -605,25 +614,30 @@ private function getDescription($variation, KeyValue $settings):string
 
     /**
      * Get variation attributes.
+	 *
      * @param array $variation
      * @param KeyValue $settings
-     * @return array<string,string>
+     * @return array
      */
     private function getVariationAttributes($variation, KeyValue $settings):array
     {
         $list = [];
         $variationAttributes = [];
 
-        foreach($variation['data']['attributes'] as $attributeValue)
-        {
-            if(array_key_exists($attributeValue['attributeId'], $this->linkedAttributeList) && strlen($this->linkedAttributeList[$attributeValue['attributeId']]) > 0)
-            {
-                if (strlen($this->attributeValueCache[$attributeValue['valueId']]) > 0)
-                {
-                    $variationAttributes[$this->linkedAttributeList[$attributeValue['attributeId']]][] = $this->attributeValueCache[$attributeValue['valueId']];
-                }
-            }
-        }
+        if(!is_null($variation['data']['attributes']))
```

Comment:
null check an der falschen stelle, durch den mutator kann $variation['data']['attributes'] nicht null sein, es muss geprüft werden ob die values in dem array null sind